Documentation/package-dependency-flow.md

```diff
@@ -42,8 +42,8 @@ ability to service the product.
 
 ## Package Versions
 
-Package dependencies that defined using 
-[Arcade's Darc patterns](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md)
+Package dependencies defined using 
+[Arcade's dependency patterns](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md)
```

Comment:
I missed committing this change in my last doc update so I am including it here.